test/test.go

```diff
@@ -28,6 +28,80 @@ func testEmbed(g abstract.Group, rand cipher.Stream, points *[]abstract.Point,
 	*points = append(*points, p)
 }
 
+func testPointSet(g abstract.Group, rand cipher.Stream) {
+	N := 1000
+	null := g.Point().Null()
+	for i := 0; i < N; i++ {
```

Comment:
I don't really get why we do this a thousand time ... If it's "just to be safe", I'm not sure I would keep it, because it only makes the test bigger and slower (it's different than the case with SDA with network communications and concurrency issues and such). If there's a specific reason, then I'd love to hear it.